src/main.ts

```diff
@@ -58,6 +58,9 @@ export type Options = {
   outputEncodeMethods: boolean;
   outputJsonMethods: boolean;
   outputClientImpl: boolean;
+  addGrpcMetadata: boolean;
```

Comment:
I have added a nestJS which encapsulate the interface only options that existed previously, it also sets the method names to "lowercase". This is needed by nestjs.

I can't encapsulate the "addGrpcmetaData" and "returnObservable" because this depends on the user.